include/kos/fs_romdisk.h

```diff
@@ -15,9 +15,27 @@
     with the genromfs program that is included in the utils portion of the tree.
 
     You can choose to automount one ROMFS image by embedding it into your binary
-    and using the appropriate KOS_INIT_FLAGS() setting. The embedded ROMFS will
-    mount itself on /rd. You can also mount additional images that you load
-    from some other source on whatever mountpoint you want.
+    and using the appropriate flags (INIT_DEFAULT by itself or INIT_FS_ROMDISK with other flags) when calling KOS_INIT_FLAGS().
+    when calling the INIT macro with a custom flag selection. The embedded ROMFS
+    will mount itself on /rd.
+
```

Comment:
IMO the examples should be based on the doc, not the other way around. It would be "more correct" if the doc says X modifications to the makefile are needed, explain why in a very brief manner and then point to an example.